ReactAndroid/src/main/java/com/facebook/react/modules/websocket/WebSocketModule.java

```diff
@@ -54,6 +54,7 @@
   private final Map<Integer, WebSocket> mWebSocketConnections = new HashMap<>();
 
   private ReactContext mReactContext;
+  private static @Nullable OkHttpClient okHttpClient;
```

Comment:
This should get cleaned up when the module is destroyed. In general, use instance variables instead of static variables with React Native modules; make this mHttpClient.